### PR TITLE
PathMarkerLayer: Support bi-directional arrows

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: src/experimental-layers/src/path-marker-layer/create-path-markers.js [hunk 1]
```diff
@@ -39,9 +37,20 @@ export default function createPathMarkers({
...
-      markers.push(marker);
+
+      if (direction.forward) {
+        const marker = createMarkerAlongPath(
+          {path: vPoints, percentage, lineLength, color, object, projectFlat});
+        markers.push(marker);
+      }
+
+      if (direction.backward) {
+        vPoints.reverse();
```
Comment (georgios-uber) [inline src/experimental-layers/src/path-marker-layer/create-path-markers.js]: This will cause an issue on multiple percentages.

Comment (ibgreen) [reply src/experimental-layers/src/path-marker-layer/create-path-markers.js]: Maybe leave a comment/TODO?

Comment (georgios-uber) [reply src/experimental-layers/src/path-marker-layer/create-path-markers.js]: I have a fix.

File: src/experimental-layers/src/path-marker-layer/create-path-markers.js [hunk 2]
```diff
@@ -10,11 +10,12 @@ function getLineLength(vPoints) {
 }
 
 const DEFAULT_COLOR = [0, 0, 0, 255];
+const DEFAULT_DIRECTION = {forward: false, backward: false};
```
[the remark this answers is not in the document]
Comment (georgios-uber) [reply src/experimental-layers/src/path-marker-layer/create-path-markers.js]: Will do.

**Pull request merged**